x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -278,6 +280,20 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
   );
 
   const singleQueryDetails = useMemo(() => liveQueryDetails?.queries?.[0], [liveQueryDetails]);
+  const liveQueryActionId = useMemo(() => liveQueryDetails?.action_id, [liveQueryDetails]);
+
+  const addToCaseButton = useCallback(
+    (payload) => (
+      <AddToCaseButton
+        queryId={payload.queryId}
+        agentIds={agentIds}
+        actionId={liveQueryActionId || ''}
```

Comment:
How likely is it for the `liveQueryActionId` to be undefined? aka setting `actionId` to an empty string? Should we disable the button if it undefined?

I think I tracked the code and it'll just result in the live query not running so maybe we'll just show an empty component within cases.

Reply:
You're right, empty string doesn't make sense. I'll add a check if we have livequeryActionId before rendering Button. We should have it after running a query, before getting results - I would risk a statement that it's obligatory.